test/app/protractorConf.js

```diff
@@ -28,34 +28,41 @@ exports.config = {
 
     browser.driver.manage().window().maximize();
 
-    if (process.env.TEAMCITY_VERSION) {
-      var jasmineReporters = require('jasmine-reporters');
-      jasmine.getEnv().addReporter(new jasmineReporters.TeamCityReporter());
-    } else {
-      var SpecReporter = require('jasmine-spec-reporter').SpecReporter;
-      jasmine.getEnv().addReporter(new SpecReporter({
-        spec: {
-          displayStacktrace: true
-        }
-      }));
-      /*
-      jasmine.getEnv().addReporter(new jasmineReporters.TerminalReporter({
-        verbosity: browser.params.verbosity, // [0 to 3, jasmine default 2]
-        color: true,
-        showStack: true
-      }));
-      */
-      var pauseOnFailure = {
-        specDone: function (spec) {
-          if (spec.status === 'failed') {
-            debugger;
-          }
-        }
-      };
+    var jasmineReporters = require('jasmine-reporters');
+    if (process.env.GITHUB_ACTIONS) {
+      // https://github.com/angular/protractor-cookbook/tree/master/jasmine-junit-reports
+      var junitReporter = new jasmineReporters.JUnitXmlReporter({
+        savePath: 'e2e-output/',
+        consolidateAll: true
+
+        // results written to file: e2e-output/junitresults.xml
 
-      // Uncomment to pause tests on first failure
-      // jasmine.getEnv().addReporter(pauseOnFailure);
+      });
+      jasmine.getEnv().addReporter(junitReporter);
     }
+    var SpecReporter = require('jasmine-spec-reporter').SpecReporter;
+    jasmine.getEnv().addReporter(new SpecReporter({
+      spec: {
+        displayStacktrace: true
+      }
+    }));
+    /*
+    jasmine.getEnv().addReporter(new jasmineReporters.TerminalReporter({
```

Comment:
any reason to hang onto this commented code anymore?

Reply:
I deleted the TerminalReporter code - not needed, thanks.  The other function is potentially useful for debugging e2e tests